src/patternfly/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -109,6 +115,7 @@ cssPrefix: pf-c-description-list
 | `.pf-c-description-list__term` | `<dt>` | Initiates a description list component term. **Required** |
 | `.pf-c-description-list__description` | `<dd>` | Initiates a description list component description. **Required** |
 | `.pf-c-description-list__text` | `<span>`, `<div>` | Initiates a description list component text element. Use a `<span>` when a child of `.pf-c-description-list__term`. **Required** |
+| `.pf-m-help-text` | `.pf-c-description-list__text` | Modifies the description list component text to be clickable. |
```

Comment:
Oh just noticed this. Maybe something more help text related in the description, like "Modifies the description list component text to indicate there is more info available for the term text." Or less wordy but less consistent with the others, "Indicates there is more info available for the description list component term text."